args/args.go

```diff
@@ -424,6 +425,11 @@ func (args *Args) setCommandLineArgs() (err error) {
 		&args.SwapFileSize, "swap-file-size", args.SwapFileSize, "Size of the swapfile; <size>[B|K|M|G]",
 	)
 
+	flag.BoolVar(
+		&args.ForceInstall, "force-install",
```

Comment:
Please rename this to 'force-destructive'. We are always installing, but I want this to reflect what specifically they are forcing an override for.

Reply:
I was thinking it would be too big `force-destructive`...but code completions this should become easier

Reply:
Done